api/src/main/java/uk/gov/hmcts/bar/api/controllers/payment/PaymentInstructionController.java

```diff
@@ -359,10 +391,21 @@ public int getUnallocatedPayment(@PathVariable("id") Integer paymentId){
     @ResponseStatus(HttpStatus.OK)
     @GetMapping("/payment-stats")
```

Comment:
do all changes to new method and endpoint so that it doesn't break the existing functionality

Reply:
GET /users/pi-stats?status=PA or RDM